telemetry/csharp/AwsToolkit.Telemetry.Events/Core/BaseTelemetryEvent.cs

```diff
@@ -18,11 +18,41 @@ public abstract class BaseTelemetryEvent
         public bool Passive = false;
 
         /// <summary>
-        /// Optional - The reason for a metric or exception depending on context
+        /// Optional - The reason for a metric or exception depending on context. It describes a certain theme of errors usually the exception class name eg. FileIOException
         /// This is often used in failure scenarios to provide additional details about why something failed.
         /// </summary>
         public string Reason;
 
+        /// <summary>
```

Comment:
Can you please update https://github.com/aws/aws-toolkit-common/blob/main/telemetry/telemetryformat.md#global-arguments with these new global fields?

Reply:
Done : https://github.com/aws/aws-toolkit-common/pull/769